### ISSUE TITLE
Encoding of property files

### ISSUE THREAD

**Issue by stritti** [ghys/habot#4]
In German the bot send answers with incorrect encoding for special chars like ä, ö, ü or ß.

For example:
"Ich entschuldige mich, ich fÃ¼rchte, ich habe keine Ahnung."

See also: https://github.com/eclipse/smarthome/issues/2639

**Comment by ghys** [comment]
Yes I saw that - I fixed it in https://github.com/ghys/habot/pull/3/commits/8b95da00fb844494059b8880d518503f84466799 in #3.
Maybe your editor converts in back to UTF-8 when you save?
